scripts/Muon/GUI/Common/plot_widget/plot_widget_model.py

```diff
@@ -146,7 +146,7 @@ def _generate_run_indices(workspace_list):
         return indices
 
     @staticmethod
-    def get_fit_workspace_and_indices(fit):
+    def get_fit_workspace_and_indices(fit,plot_diff = True):
```

Comment:
Maybe change the argument name to `with_diff` or something along those lines - as `plot_diff` doesn't have much context here.